src/Test.FunctionalTests.BinSkim.Driver/BaselineTestData/Expected/Binskim.linux-x64.dll.sarif

```diff
@@ -96,6 +96,8 @@
       "tool": {
         "driver": {
           "name": "testhost",
+          "organization": "Microsoft Corporation",
```

Comment:
>Microsoft [](http://example.com/codeflow?start=27&length=9)

OK, so what actually appears to have happened here is that you copied over test output from our literal test run?

Instead, try doing this from a console window (starting at the root of your BinSkim enlistment):

cd .\src\Test.FunctionalTests.BinSkim.Driver
UpdateBaselines.cmd

This should undo most of these changes once you commit them.

What I did was to enlist clean in main, run baseline update and confirmed the checked in files are good.

We could try to be a bit more friendly and rewrite the test output so that it doesn't have the deltas that burned you, that would be a nice tech debt clean-up later...